packages/lesswrong/components/posts/PostsNewForm.tsx

```diff
@@ -18,7 +18,7 @@ import type { PostSubmitProps } from './PostSubmit';
 // Also used by PostsEditForm
 export const styles = (theme: ThemeType): JssStyles => ({
   postForm: {
-    width:715,
+    maxWidth: 715,
```

Comment:
I believe this is just incorrect on prod, since we want the form to be responsive between 715px and `xs`